examples/prometheus/prometheus.yaml

```diff
@@ -286,3 +374,76 @@ objects:
         - source_labels: [__meta_kubernetes_service_name]
           action: replace
           target_label: kubernetes_name
+
+      alerting:
+        alertmanagers:
+        - scheme: http
+          static_configs:
+          - targets:
+            - "localhost:9093"
+
+# Create a fully end-to-end TLS connection to the proxy
+- apiVersion: route.openshift.io/v1
```

Comment:
Comment here about this being exposed for external use if so (clients who wish to grab alter status).